code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1182,8 +1182,15 @@
 		return
 
 	var/obj/effect/decal/cleanable/greenglow/glow = exposed_turf.spawn_unique_cleanable(/obj/effect/decal/cleanable/greenglow)
-	if(!QDELETED(glow))
-		glow.reagents.add_reagent(type, reac_volume)
+	if(!glow)
```

Comment:
Any reason why this isn't using qdeleted still?

Reply:
good catch, changed it because ants didn't do that but reading Initialize for cleanable it's possible for it to qdel in init so will add it to both